tests/langchain/test_langchain_databricks_dependency_extraction.py

```diff
@@ -35,6 +35,12 @@ def __init__(
         self.task = task
 
 
+def remove_langchain_modules():
+    langchain_modules = [k for k in sys.modules.keys() if k.startswith("langchain.")]
```

Comment:
```suggestion
    langchain_modules = [k for k in sys.modules.keys() if k.startswith("langchain." or k.startswith("langchain_community.")]
```
Shall we also remove
```
import langchain_community.*
```

`langchain_community` might also introduce the lazy loaded modules.

Reply:
I was getting some pydantic errors based on that, so fixed it using a different approach. Let me know if this makes sense.